pkgs/applications/window-managers/wtftw/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchgit, rustPlatform, cargo, libXinerama, libX11, xlibs, pkgconfig }:
+
+with import <nixpkgs> { }; with rustPlatform; with xlibs; with stdenv;
```

Comment:
removing `'with import...'` and replacing it with `with rustPlatform;` (which appears to be standard behavior for packages using the rust build system) results in the following error:

 nix-build -A wtftw --check --show-trace
> error: while evaluating the attribute ‘nativeBuildInputs’ of the derivation ‘wtftw’ at /home/cor/devel/nixpkgs/pkgs/applications/window-managers/wtftw/default.nix:7:3:
> cannot coerce a set to a string, at /home/cor/devel/nixpkgs/pkgs/applications/window-managers/wtftw/default.nix:7:3

I am very new to this, so apologies if I'm missing something simple...